Assets/MRTK/SDK/Experimental/InteractiveElement+StateVisualizer/InteractiveElement/BaseInteractiveElement.cs

```diff
@@ -24,6 +24,7 @@ public abstract class BaseInteractiveElement :
         IMixedRealityPointerHandler,
         IMixedRealitySpeechHandler
     {
+        [Experimental]
```

Comment:
Was this change intentional in this PR? Reminder for if you move BaseInteractiveElement to experimental: move it to the experimental folder, and experimental namespaces as well

Reply:
Reverted the changes and will modify in a separate PR.